netebpfext/user/fw_thunk.h

```diff
@@ -0,0 +1,863 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "kernel_thunk.h"
+#include "ndis_thunk.h"
+
+typedef void* FWPS_CALLOUT_CLASSIFY_FN;
```

Comment:
Question regarding this file: is there no way to wrap including "fwpvi.h" or "fwpmk.h" or whatever in a way that we can include them from in here without duplicating them?

Reply:
I don't see any simple path to achieving this, unfortunately. A project either uses the user mode or the kernel mode SDK and these are simply not present in the kernel mode SDK. There doesn't seem to be a way to easily reference the kernel SDK from a user mode project.